src/page-objects/InternetIdentityPage.ts

```diff
@@ -109,8 +113,10 @@ export class InternetIdentityPage {
     await iiPage.locator('#registerButton').click();
     await iiPage.locator('[data-action=construct-identity]').click();
 
-    await iiPage.locator('input#captchaInput').fill('a');
-    await iiPage.locator('#confirmRegisterButton').click();
+    if (params?.captcha === true) {
```

Comment:
Why not `params?.captcha`?

Reply:
To express the fact that it can be nullish given that we generally use `isNullish` but I can change it if you want?

Reply:
I'll merge it given that you already approved.

Reply:
Yes, I thuoght it was because of that, but I wasn't sure.